Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Race Genes

## Changelog:
- Your race trait depends on the ethnicity you pick in the Ruler Designer. Previously it was based on your start culture.


## Developer changelog:
- Renamed genes that set static models to murlocs, furbolgs, etc.
- Added `gene_race` that used to assign the race trait in the Ruler Designer.
- Removed many triggers from `wc_race_triggers.txt` since their function was handled by `is_race_trigger`.
- Removed `potential_race_<race>_trigger`'s since their function was handled by `is_race_trigger`.
- Renamed `<race>_portrait_gfx` to `creature_<race>_gfx>` to fit the style of the race traits.
- Removed `moarg` culture because it wasn't used.
- Removed these race traits:
  - `creature_blood_elf` (because there's blood elf culture)
  - `creature_fel_orc` (because fel orc is fel-infused orc)
  - `creature_eredar` (but `creature_draenei` is called Eredar now since both the draenei and man'ari are the same race)
  - `creature_iron_dwarf` (because it wasn't used and not worth it)
  - `creature_earthen` (because it wasn't used and not worth it)


# How to test:
Try to create different races in the Ruler Designer and check what race trait is assigned.
